composer.json

```diff
@@ -285,6 +287,9 @@
                 "https://www.drupal.org/project/redirect/issues/3057250": "https://www.drupal.org/files/issues/2024-08-11/redirect--2024-08-11--3057250-79.patch",
                 "https://www.drupal.org/project/redirect/issues/3018897": "https://www.drupal.org/files/issues/2024-08-12/redirect-3018897-28.patch"
             },
+            "drupal/role_description": {
+                "https://www.drupal.org/project/role_description/issues/3476522": "https://git.drupalcode.org/project/role_description/-/merge_requests/2.patch"
```

Comment:
Can you please download the patch instead of using the merge request link. See this comment in a different ticket: https://github.com/SU-HSDO/suhumsci/pull/1813#discussion_r2067176802